### DISCUSSION TITLE
Remove webpack5 to dependencies - no magnificent change after building the app

### DISCUSSION THREAD

**Discussion by herol3oy** [vercel/next.js#26818] [Help]
I found `webpack5` installed in a **NextJS** app under **package.json** but then I removed it since webpack5 is already a built-in feature in NextJS apps. However, I didn't find any magnificent change after building the app? For example I thought the bundle size would be larger but after building, it was the same!! I wonder if somebody can comment on this!??

**Comment by robertpiosik** [comment]
Did you enable `webpack5` in the `next.config.js`?

**Reply by herol3oy** [reply]
There's no configuration about `webpack5` in `next.config.js` The version of app's NextJS is 10.0.7. Isn't it `webpack5` by default? Do I need to enable it in the `next.config.js`?

**Reply by bitroy** [reply]
To enable webpack5 in Next 10.0.7
Add this in next.config.js
```
module.exports = {
  future: {
    webpack5: true,
  },
};
```

**Reply by timneutkens** [reply]
I'd highly recommend upgrading to Next.js 11 where it's enabled by default.

**Reply by herol3oy** [reply]
@timneutkens For sure, I will. But any comment on why when webpack5 was installed as dependency I couldn't see any magnificent change in my built app bundle size? Was it override the original built in webpack5?
